capa/render/default.py

```diff
@@ -71,7 +107,12 @@ def rec(match: rd.Match):
     return matches
 
 
-def render_capabilities(doc: rd.ResultDocument, ostream: StringIO):
+def render_rule_name(name: str) -> str:
+    url = f"https://mandiant.github.io/capa/rules/{urllib.parse.quote(name)}/"
```

Comment:
Clicking the hrefs in the CLI results in 404s:
Current: `https://mandiant.github.io/capa/rules/forwarded export/`
Expected: `https://mandiant.github.io/capa/rules/forwarded-export.html`

I understand this is wip, so just pointing this out for now

Reply:
This PR updates the paths that the website will accept. It won't work with the URLs on the website.

Reply:
Ah, I see

Reply:
links work now that the website is updated.